### PR TITLE
156500836 update port map with buttons

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* Buttons to add port
   
# Fixed
* Swedish translations

### CODE REVIEW

File: ote/src/cljs/ote/views/route/stop_sequence.cljs [hunk 1]
```diff
@@ -61,82 +61,106 @@
...
-    (fn [this]
-      (let [deleting? (atom false)
-            ^js/L.Map
-            m (aget this "refs" "stops-map" "leafletElement")]
+  (let [dc (atom nil)]
+    (r/create-class
+      {:component-did-mount
+        (fn [this]
+          (let [deleting? (atom false)
+               ^js/L.Maps
```
Comment (tatut) [inline ote/src/cljs/ote/views/route/stop_sequence.cljs]: extra `s` in type, `L.Map` -> `L.Maps` mistake?

File: ote/src/cljs/ote/views/route/stop_sequence.cljs [hunk 2]
```diff
@@ -61,82 +61,106 @@
...
+              :add-features?           true
+              :localization            {:toolbar  {:buttons {:marker (tr [:route-wizard-page :stop-sequence-leaflet-button-marker])}}
+                                        :handlers {:marker {:tooltip {:start (tr [:route-wizard-page :stop-sequence-leaflet-button-start])}}}}})))
+            :component-will-receive-props (fn [this [_ _ route]]
+                                       (let [^js/L.map m (aget this "refs" "stops-map" "leafletElement")]
+                                         (if (get-in route [:map-controls :show?])
+                                           (.addControl m @dc)
+                                           (.removeControl m @dc))))
+            :reagent-render
+                                     (fn [e! route]
```
Comment (tatut) [inline ote/src/cljs/ote/views/route/stop_sequence.cljs]: Indentation seems of in render fn.

File: ote/src/cljs/ote/views/route/stop_sequence.cljs [hunk 3]
```diff
@@ -61,82 +61,106 @@
...
 
-        (when-let [stop-sequence (seq (::transit/stops route))]
-          [leaflet/Polyline
-           {:positions (clj->js (mapv (comp flip-coords ::transit/location) stop-sequence))
-            :color "red"}])]])}))
-
-(defn- route-stops [e! stop-sequence]
+(defn- route-stops [e! {stop-sequence ::transit/stops :as route}]
   [:div {:style {:width "30%" :margin "1em"}}
+   (when (not (get-in route [:map-controls :show?]))
```
Comment (tatut) [inline ote/src/cljs/ote/views/route/stop_sequence.cljs]: Indent body of `when`

File: ote/src/cljs/ote/views/route/stop_sequence.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/route/stop_sequence.cljs]: yep. thanx

File: ote/src/cljs/ote/views/route/stop_sequence.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/route/stop_sequence.cljs]: My idea likes to do intentions a bit weirdly.

**Comment by rckopteri** [comment]
Nice catch Fox. Fixed now.

**Pull request merged**